docs/eventlisteners.md

```diff
@@ -257,7 +263,39 @@ spec:
 
 With the help of `kubernetesResource` user can specify [PodTemplateSpec](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3704).
 
-Right now the allowed values as part of `podSpec` are
+#### CustomResource
+`CustomResource` have one field which supports dynamic object
+* runtime.RawExtension
+
+Used Knative Service as an example to demonstrate usage of `CustomResource`
+
+**Note:** `Knative Should be installed on the cluster` [ref](https://github.com/tektoncd/community/blob/main/teps/0008-support-knative-service-for-triggers-eventlistener-pod.md#note)
+```yaml
+spec:
+  resources:
+    customResource:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      metadata:
+        name: knativeservice
+      spec:
+        template:
+          spec:
+            serviceAccountName: tekton-triggers-example-sa
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+```
+
+With the help of `CustomResource` user can specify any dynamic object which follows this [Contract](https://github.com/tektoncd/community/blob/main/teps/0008-support-knative-service-for-triggers-eventlistener-pod.md#contract).
```

Comment:
Should we move the contract to the triggers docs instead of pointing to the TEP?

Reply:
Modified doc